src/families/bitcoin/js-synchronisation.ts

```diff
@@ -323,17 +323,18 @@ const getAccountShape: GetAccountShape = async (info) => {
   const { txs: transactions } = await wallet.getAccountTransactions(
     walletAccount
   );
+
+  const accountAddresses: Set<string> = new Set<string>();
   const accountAddressesWithInfo = await walletAccount.xpub.getXpubAddresses();
-  const accountAddresses = accountAddressesWithInfo
-    ? accountAddressesWithInfo.map((a) => a.address)
-    : [];
+  accountAddressesWithInfo.map((a) => accountAddresses.add(a.address));
```

Comment:
maybe
const accountAddresses = new Set( accountAddresses.map(  a => a.address));
would be more concise.